BrazhnikovaOV/lab_3/main.cpp

```diff
@@ -0,0 +1,209 @@
+#include<iostream>
+using namespace std;
+class Dynamic_massive {
+private:
+	int cap = 0;
+	int* Newarr; 
+	int size;
+	int i = 0;
+public:
+	Dynamic_massive(int nsize = 0) {
+		size = 0;
+		if (nsize == 0) {
+			cap = 2;
+		}
+		else {
+			cap = nsize * 2;
+		delete[] Newarr;
```

Comment:
1) What's the point of deleting Newarr in constructor?
2) Since Newarr is not initialized, this line results in undefined behavior